test/common/PrepDataFuncs.cpp

```diff
@@ -339,4 +340,35 @@ namespace RcclUnitTesting
     }
     return TEST_SUCCESS;
   }
+
+  ErrCode DefaultPrepData_Send(CollectiveArgs &collArgs)
+  {
+
+    CHECK_CALL(CheckAllocation(collArgs));
+    if (collArgs.numInputElements != collArgs.numOutputElements)
+    {
+      ERROR("Number of input elements must match number of output elements for Broadcast\n");
+      return TEST_FAIL;
+    }
+    size_t const numBytes = collArgs.numInputElements * DataTypeToBytes(collArgs.dataType);
+      return collArgs.inputGpu.FillPatternSR(collArgs.dataType,
+                                               collArgs.numInputElements,
+                                               collArgs.globalRank, true);
+  }
+
+  ErrCode DefaultPrepData_Recv(CollectiveArgs &collArgs)
+  {
+
+    CHECK_CALL(CheckAllocation(collArgs));
+    if (collArgs.numInputElements != collArgs.numOutputElements)
+    {
+      ERROR("Number of input elements must match number of output elements for Broadcast\n");
+      return TEST_FAIL;
+    }
+    size_t const numBytes = collArgs.numInputElements * DataTypeToBytes(collArgs.dataType);
+    return collArgs.expected.FillPatternSR(collArgs.dataType,
```

Comment:
Doesn't seem to be any particular reason to use a special FillPatternSR - The standard one works here for now.